### PR TITLE
[14.0][UPT] template from copier

### LINKED ISSUE #505: repo readme.md
Hi, would it be possible to fix the readme.md for this repo to align it to the other repos? this is how 14 appears:

image

I also see in v12 and 13 there are still links to transifex

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Fixes https://github.com/OCA/operating-unit/issues/505

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+
```
Comment (MiquelRForgeFlow) [inline .pylintrc:2]: wrong

File: .gitignore [hunk 2]
```diff
@@ -15,7 +15,6 @@ build/
 develop-eggs/
 dist/
 eggs/
-lib/
```
Comment (MiquelRForgeFlow) [inline .gitignore:18]: wrong

File: .pylintrc (hunk 1 above)
Comment (AaronHForgeFlow) [reply .pylintrc:2]: why? I just did copier update, it should work like this, isn't it?

Comment (MiquelRForgeFlow) [reply .pylintrc:2]: yes, but it's ugly having double space

Comment (AaronHForgeFlow) [reply .pylintrc:2]: that's bad

**Comment by Saran440** [comment]
@AaronHForgeFlow Hello, This PR conflict, do you still do this PR or will move it to https://github.com/OCA/operating-unit/pull/518

**Comment by AaronHForgeFlow** [comment]
Hi @Saran440  We can follow up in your PR

**Pull request closed without merging**